Dockerfile

```diff
@@ -49,5 +49,9 @@ RUN set -x \
 RUN set -x \
 	&& eval "GOARCH=ppc64le go build $BUILD_FLAGS -o /go/bin/gosu-ppc64el" \
 	&& file /go/bin/gosu-ppc64el
+RUN set -x \
+		&& eval "GOARCH=s390x go build $BUILD_FLAGS -o /go/bin/gosu-s390x" \
+		&& file /go/bin/gosu-s390x
+
 
```

Comment:
Can you modify the whitespace to match the rest of the `RUN` instructions and delete this extra newline? (should only be a single newline in this space)